### PR TITLE
FlashMessages that animate!

### PR DESCRIPTION
**Pull request by patrickkim**

### CODE REVIEW

File: src/atoms/FlashMessage/flash_message.module.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  100% {
+    max-height: 200px;
+    opacity: 1;
+  }
+}
+
+.fadeIn { animation: fadeIn 600ms ease-in both; }
+.fadeInDown { animation: fadeInDown 600ms cubic-bezier(0.175, 0.885, 0.32, 1.275) both; }
+.slideDown { animation: slideDown 1000ms cubic-bezier(0.175, 0.885, 0.32, 1.275) both; }
```
Comment (drewdrewthis) [inline src/atoms/FlashMessage/flash_message.module.scss]: @patrickkim We probably don't want to have to write this every time, and these should probably be standardized. I suggest importing keyframes into the global file and then having placeholders that can be used:

```
.fadeIn {
   @extend %fadeIn;
}
```

File: src/atoms/FlashMessage/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import styles from './flash_message.module.scss';
+
+function FlashMessage(props) {
+  return (
+    <div className={classnames(styles['flash-message'], styles[props.variant])}>
```
Comment (drewdrewthis) [inline src/atoms/FlashMessage/index.js]: @patrickkim Why did you break away from our convention of deconstructing the `props` here? I know it's only two, but it's nicer if someone wants to add a third down the line.

File: src/molecules/asides/ImageAside/image-aside.module.scss [hunk 3]
```diff
@@ -100,7 +100,9 @@ $aside-text-color:               color('neutral-2');
 }
 
 .small {
+  display: flex;
   align-items: center;
+  margin-left: 0;
```
Comment (drewdrewthis) [inline src/molecules/asides/ImageAside/image-aside.module.scss]: Where was this margin coming from?

File: styleguide.config.js [hunk 4]
```diff
@@ -46,14 +46,22 @@ module.exports = {
...
+          name: 'FlashMessage',
+          components: 'src/molecules/FlashMessage/**/*.js'
+        },
+        {
+          name: 'FormFields',
+          components: 'src/molecules/formfields/**/*.js'
+        },
+        {
+          name: 'Headers',
+          components: () => [ 'src/molecules/PriceDiscountHeader/index.js' ]
```
Comment (drewdrewthis) [inline styleguide.config.js]: We shouldn't need to explicitly write these in every time. They should just get brought in by the "Other" category.

File: src/atoms/FlashMessage/Readme.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+```example
+  <div>
+    <Button onClick={() => setState({ showMessage: !state.showMessage })}>Click me!</Button>
+    <Spacer xsmall />
+    { state.showMessage &&
+      <FlashMessage variant='fadeInDown'>
+        <ImageAside
+          small
+          basic={false}
```
Comment (drewdrewthis) [inline src/atoms/FlashMessage/Readme.md]: @patrickkim We shouldn't have to do this. I made a ticket to refactor this: https://app.clubhouse.io/policygenius/story/2216/refactor-image-aside

File: src/atoms/FlashMessage/index.js (hunk 2 above)
Comment (patrickkim) [inline src/atoms/FlashMessage/index.js]: No reason, force of habit. Destructuring / es6 is still pretty new.

File: src/atoms/FlashMessage/flash_message.module.scss (hunk 1 above)
Comment (patrickkim) [inline src/atoms/FlashMessage/flash_message.module.scss]: Maybe when theres more than 1 component with animations that would be worth doing. Didn't want to over-engineer a global standard without a reference to other use cases.

File: styleguide.config.js (hunk 4 above)
Comment (patrickkim) [inline styleguide.config.js]: 💯

File: src/atoms/FlashMessage/Readme.md (hunk 5 above)
Comment (patrickkim) [inline src/atoms/FlashMessage/Readme.md]: 👍

File: src/molecules/asides/ImageAside/image-aside.module.scss (hunk 3 above)
Comment (patrickkim) [inline src/molecules/asides/ImageAside/image-aside.module.scss]: No idea... killed it.

**Pull request merged**